allensdk/brain_observatory/behavior/behavior_ophys_experiment.py

```diff
@@ -327,6 +328,28 @@ def _get_motion_correction():
             date_of_acquisition=behavior_session._date_of_acquisition
         )
 
+    def update_targeted_imaging_depth(self, ophys_experiment_ids: List[int]):
+        """Update the value for targeted imaging depth given a set of
+        experiments to be published.
+
+        Compute the targeted_imaging_depth (average over experiments in a
+        container) only for those experiments input.
+
+        Parameters
+        ----------
+        ophys_experiment_ids : list of ints
+            Set of experiments to calculate targeted_imaging_depth for. Needs
```

Comment:
Slightly unclear. "Set of experiments to use to calculate the targeted_imaging_depth for this experiment"

Reply:
Changed to subset and mentioned the container.